ti/instrumentation/java/bazel.go

```diff
@@ -59,19 +101,22 @@ func (b *bazelRunner) GetCmd(ctx context.Context, tests []ti.RunnableTest, userA
 	clss := []string{}
 	set := make(map[string]interface{})
 	ut := []string{}
+	rls := []string{}
 	for _, t := range tests {
-		if _, ok := set[t.Class]; ok {
```

Comment:
why do we want to skip this?

Reply:
I saw an instance in harness-core repo where package.class was repeated but module was different. This can be the case for any repo, hence had to remove this check.